src/client/components/ProfileCareerResult/ProfileCareerResult.component.js

```diff
@@ -75,13 +75,14 @@ export default function ProfileCareerResult({ jobTitle }) {
             <p key={content.contentHead}>
               <strong>{content.contentHead}</strong>
               <span>{content.content}</span>
+              <a href={`/career/${jobTitle}`}>
+                <button type="button" className="triangle">
+                  {' '}
+                </button>
+              </a>
             </p>
```

Comment:
Hi Nashmil. I think, the triangle button only appears once for each career. It means here see more, right? So it only comes once at the last of the paragraph. As you have used it inside map, I think you need to move this outside so that after the paragraphs are rendered then to see more this button comes at last. What do you think?